kubernetes/chart/values.yaml

```diff
@@ -0,0 +1,112 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: akkeris/region-api
+  tag: release-27
```

Comment:
Putting the tag in here seems awkward, we won't know the tag version until after the commit to master, so we'll always have to be doing two commits to master.  The first to create the new release with the code, and the second to update the manifests.

Reply:
then we should use latest here. The point is to copy the values.yaml and only use it as an example.